src/frameworks/uikit/ui_view.rs

```diff
@@ -67,7 +67,26 @@ pub const CLASSES: ClassExports = objc_classes! {
     env.objc.get_known_class("CALayer", &mut env.mem)
 }
 
-// TODO: initWithFrame:, accessors, etc
+- (id)initWithFrame:(CGRect)frame { // CGRect
+    let host_object: &mut UIViewHostObject = env.objc.borrow_mut(this);
+    host_object.bounds = frame;
+    host_object.center = CGPoint { x: frame.size.width / 2.0, y: frame.size.height / 2.0 };
```

Comment:
FYI, from what I remember of the docs for `bounds`, `center` and `frame`, I think this is wrong:

* The center will have to be offset by the origin in the frame also
* The origin in the bounds is probably 0, at the very least it's not the same as the origin of the frame

I also already have some code to handle this, so I can clean up and push that instead if you want.

Though on that note, I'm not perfectly sure my own understanding is correct.